advanced_functionality/pytorch_yolov5_training_and_hpo/transfer_learning_and_hpo_yolov5.ipynb

```diff
@@ -0,0 +1,1379 @@
+{
```

Comment:
<p>Stick with present tense -&gt; "This notebooks takes you...", "You use AWS Deep Learning Containers...", "You also use Amazon SageMaker's Automated Model Tuning"</p><p> </p><p>This applies to everything in this notebook as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3601/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Changed all to present tense. Also stayed with second person singular. Do let me know if this is OK.</p>

Reply:
<p>Yeah, it looks good</p>